test/core/tsi/ssl_transport_security_utils_test.cc

```diff
@@ -429,6 +445,135 @@ INSTANTIATE_TEST_SUITE_P(FrameProtectorUtil, FlowTest,
 
 #endif  // OPENSSL_IS_BORINGSSL
 
+X509_CRL* read_crl(absl::string_view crl_string) {
+  BIO* crl_bio =
+      BIO_new_mem_buf(crl_string.data(), static_cast<int>(crl_string.size()));
+  // Errors on BIO
+  if (crl_bio == nullptr) {
+    return nullptr;
+  }
+  X509_CRL* crl = PEM_read_bio_X509_CRL(crl_bio, nullptr, nullptr, nullptr);
+  BIO_free(crl_bio);
+  return crl;
+}
+
+X509* read_cert(absl::string_view cert_string) {
+  BIO* cert_bio =
+      BIO_new_mem_buf(cert_string.data(), static_cast<int>(cert_string.size()));
+  // Errors on BIO
+  if (cert_bio == nullptr) {
+    return nullptr;
+  }
+  X509* cert = PEM_read_bio_X509(cert_bio, nullptr, nullptr, nullptr);
+  BIO_free(cert_bio);
+  return cert;
+}
+
+TEST(CrlUtils, VerifySignatureValid) {
+  absl::StatusOr<Slice> crl_slice = LoadFile(kValidCrl, false);
+  ASSERT_TRUE(crl_slice.ok()) << crl_slice.status();
```

Comment:
optional: Consider using `ASSERT_EQ(crl_slice.status(), absl::OkStatus());` so that we get more information on a failure. Similarly throughout.

Reply:
Done